output/json.go

```diff
@@ -45,3 +47,7 @@ func (j *JSON) Output(results []CheckResult) error {
 	fmt.Fprintln(j.Writer, out.String())
 	return nil
 }
+
+func (j *JSON) Report(results []*tester.Result) error {
+	return nil
```

Comment:
nit: Since this output isn't supported and because this path shouldn't actually be hit, I think it would be better to return an error here.